test/controllers/SecondContactHavePhoneControllerSpec.scala

```diff
@@ -37,7 +37,7 @@ class SecondContactHavePhoneControllerSpec extends SpecBase with MockitoSugar {
   override def onwardRoute = Call("GET", "/foo")
 
   val formProvider = new SecondContactHavePhoneFormProvider()
-  val form         = formProvider()
+  val form         = formProvider("clientSecondContactHavePhone")
```

Comment:
This should just be "secondContactHavePhone" because this controller is for the organisation journey